hnn_core/tests/test_gui.py

```diff
@@ -164,6 +165,13 @@ def test_gui_upload_drives():
     gui._simulate_upload_connectivity(file1_url)
     assert len(gui.drive_widgets) == 1
 
+    # Load file with more drives and make sure it's in the right order
+    gui.delete_drive_button.click()
+    gui._simulate_upload_drives(file3_url)
+    drive_names = [widget['name'] for widget in gui.drive_widgets]
+    assert drive_names == ['evdist1', 'evprox1', 'evprox2',
```

Comment:
One test that comes to mind is 
1) add drives through the GUI
2) export to file
3) load back in and ensure order matches

I'm assuming this isn't possible until the load button is actually implemented?

Reply:
I think that test would be more appropriate for an export test. Since it's really testing that the gui is able to write a file with the correct structure. This one is testing that the file upload respects the order of the input file's drives.